pages/linux/brew.md

```diff
@@ -10,13 +10,13 @@
 
 `brew install {{formula}}`
 
-- List installed formulae [with matching name]:
+- List installed formulae, argument `formula` is optional:
 
-`brew list {{[text]}}`
+`brew list {{formula}}`
```

Comment:
Saying that the argument is optional doesn't handle the fact that its inclusion makes this command do something different -- so these are still two commands in one.

Please remove the parameter from this example altogether. If you think it makes sense to include it (and that it justifies bumping the page to the maximum size), add a new example with `brew list {{formula}}` and the appropriate description.